xtask/src/main.rs

```diff
@@ -25,13 +26,19 @@ fn clang_format() {
         if let Some(ext) = entry.path().extension() {
             match ext.to_str().unwrap() {
                 "h" | "c" | "cpp" | "cc" => {
-                    let res = Command::new("clang-format")
+                    let mut clang_format = ykllvm_bin_dir();
```

Comment:
It's easier to do `llvm_bin("clang-format")`. [This will also `panic` if it can't find the binary.]

Reply:
Ohhhh.

I was confused by:

> then use ykbuild::ykllvm_bin_dir("clang-format") and it'll pick the right one up automatically.

which didn't have the right args!

Reply:
Sorry!

Reply:
Try [509747c](https://github.com/ykjit/yk/pull/693/commits/509747cbe7fd82c7093e423e214481775a035117)